src/pet.js

```diff
@@ -0,0 +1,125 @@
+//magic numbers
+const MAX_FITNESS = 10;
+const MIN_HUNGER = 0;
+const HUNGRY = "I am hungry"
+const UNFIT = "I need a walk"
+const BOTH = "I am hungry AND I need a walk"
```

Comment:
I would rename this variable, as `BOTH` isn't very descriptive. something like `HUNGRY_AND_UNFIT` would be better. You also already have the `I am hungry` and `I need a walk` strings defined above, so you could use a template literal to make your code more DRY, i.e.
```
const HUNGRY_AND_UNFIT = `${HUNGRY} and ${UNFIT}`
```